microbench/src/main/java/io/netty/microbench/http2/internal/hpack/DecoderBenchmark.java

```diff
@@ -92,18 +92,15 @@ public Http2Headers add(CharSequence name, CharSequence value) {
                 return this;
             }
         };
-        decoder.decode(input.duplicate(), headers);
+        decoder.decode(0, input.duplicate(), headers);
     }
 
-    private byte[] getSerializedHeaders(List<Header> headers, boolean sensitive) {
-        Encoder encoder = new Encoder(4096);
+    private byte[] getSerializedHeaders(Http2Headers headers, boolean sensitive) throws Http2Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "sensitive". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1172)